.vscode/settings.json

```diff
@@ -28,7 +28,7 @@
   "files.exclude": {
     "**/prettier.config.cjs": true,
     "**/node_modules/**/dist": false,
-    "**/{examples,packages}/**/dist": true,
+    // "**/{examples,packages}/**/dist": true,
```

Comment:
TODO remove this -- it was preventing me from seeing dist so I could debug some setup issues